### DISCUSSION TITLE
Usability Issue Joomla 3.9.23 - Assigned Pages to Module

### DISCUSSION THREAD

**Discussion by B3nito** [joomla/joomla-cms#31550] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
I have an page with 500 menu items per languages and it is 8 languages.
Assigning Modules (and also in templates) is big hell. If you look for wrong assignment you cannot scan effectly what is checked and what not.

### Describe the solution you'd like
Maybe a side by side view. On the right side what is assigned with posibility to unassign and on left side what can be assigned?

**Comment by ghost** [comment]
New features go in J4.

**Comment by shoulders** [comment]
@gostn dumb question - how do i post a new feature for joomla 4 on github?

**Comment by ghost** [comment]
in this case i would use title ´Usability Issue - Assigned Pages to Module´ or ´[4] Usability Issue - Assigned Pages to Module´. As @B3nito use ´3.9.23´ i guessed he wants this featur in j3 so i informed him.
